todo_app.sql

```diff
@@ -13,10 +13,13 @@ CREATE TABLE tasks
 );
 
   ALTER TABLE tasks DROP completed;
-  ALTER TABLE tasks DROP updated_at;
-  ALTER TABLE tasks ADD updated_at timestamp
-  without time zone NOT NULL DEFAULT now
+  ALTER TABLE tasks ALTER COLUMN updated_at
+  SET
```

Comment:
Can you figure out how to merge these two statements together into just one statement?

Reply:
Couldn't get it to work... google wasn't proving fruitful either.  Cohort & TA didn't have immediate answer either...

Reply:
```
ALTER TABLE tasks 
  ALTER COLUMN updated_at SET NOT NULL, 
  ALTER COLUMN updated_at SET default now();
```